static/app/views/performance/browser/webVitals/pageOverview.tsx

```diff
@@ -61,13 +61,13 @@ const SAMPLES_COLUMN_ORDER: GridColumnOrder<
   keyof TransactionSampleRowWithScoreAndExtra
 >[] = [
   {key: 'user.display', width: COL_WIDTH_UNDEFINED, name: 'User'},
-  {key: 'measurements.lcp', width: COL_WIDTH_UNDEFINED, name: 'LCP'},
-  {key: 'measurements.fcp', width: COL_WIDTH_UNDEFINED, name: 'FCP'},
-  {key: 'measurements.fid', width: COL_WIDTH_UNDEFINED, name: 'FID'},
-  {key: 'measurements.cls', width: COL_WIDTH_UNDEFINED, name: 'CLS'},
-  {key: 'measurements.ttfb', width: COL_WIDTH_UNDEFINED, name: 'TTFB'},
-  {key: 'score', width: COL_WIDTH_UNDEFINED, name: 'Score'},
-  {key: 'view', width: COL_WIDTH_UNDEFINED, name: 'View'},
+  {key: 'measurements.lcp', width: 60, name: 'LCP'},
+  {key: 'measurements.fcp', width: 60, name: 'FCP'},
+  {key: 'measurements.fid', width: 60, name: 'FID'},
+  {key: 'measurements.cls', width: 60, name: 'CLS'},
+  {key: 'measurements.ttfb', width: 60, name: 'TTFB'},
+  {key: 'score', width: 60, name: 'Score'},
+  {key: 'view', width: 110, name: 'View'},
```

Comment:
should we extract these into constants?

Reply:
Hey Abhi, I restored these back to COL_WIDTH_UNDEFINED!